public/index.html

```diff
@@ -9,37 +9,167 @@
+            <div class="project-name">Slider Using Swiper API</div>
+            <a href="slider box/index.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 18</div>
+            <div class="project-name">Carousel Solar System</div>
+            <a href="carousal/index.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 19</div>
+            <div class="project-name">Planto</div>
+            <a href="plantwebsite/plant.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 20</div>
+            <div class="project-name">EveSparks</div>
+            <a href="https://evesparks.onrender.com/" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 21</div>
+            <div class="project-name">Video BG Slider Using React</div>
+            <a href="https://github.com/dhairyagothi/50_days_50_web_project/tree/Main/public/travel_website" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 22</div>
+            <div class="project-name">Page Loader</div>
+            <a href="./pageloader/pageloader.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 24</div>
+            <div class="project-name">Chat Bot</div>
+            <a href="./AI ChatBot/chatbot.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 25</div>
+            <div class="project-name">Tic-Tac-Toe</div>
+            <a href="TicTacToe/index.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 26</div>
+            <div class="project-name">Maze Game</div>
+            <a href="./Maze-Game-main/index.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 27</div>
+            <div class="project-name">Memory Game</div>
+            <a href="./MemoryGame/index.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 28</div>
+            <div class="project-name">Wordle</div>
+            <a href="./WORDLE/index.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 29</div>
+            <div class="project-name">Snake Game</div>
+            <a href="./snake_game/index.html" class="demo-link">Demo</a>
+        </div>
+        <div class="table">
+            <div class="day-number">Day 30</div>
+            <div class="project-name">Flappy-bird-game</div>
+            <a href="./Flappy-bird-main/index.html" class="demo-link">Demo</a>
+        </div>
```

Comment:
instead of adding and deleting the content of a file 
PULL the PR
this is not a good practise